### PR TITLE
fixed broken link in faq section

### PR DESCRIPTION
**Pull request by vj-codes**
The link now redirects to main readme of the project as expected.

### CODE REVIEW

**Comment by geoffromer** [comment]
This looks good, but the pre-commit is failing due to a minor formatting issue. Can you please fix that?

**Comment by vj-codes** [comment]
> This looks good, but the pre-commit is failing due to a minor formatting issue. Can you please fix that?

@geoffromer done:)

**Review by MrKrishnaAgarwal — APPROVED** [review]

File: docs/project/faq.md [hunk 1]
```diff
@@ -139,8 +139,8 @@ one.
 
 ## Why build Carbon?
 
-See the [project README](#why-build-carbon) for an overview of the motivation
-for Carbon. This section dives into specific questions in that space.
+See the [project README](/README.md#why-build-carbon) for an overview of the
```
Comment (jonmeow) [inline docs/project/faq.md]:
```suggestion
See the [project README](/README.md#why-build-carbon) for an overview of the
```

Fix end-of-line whitespace.

**Review by jonmeow — APPROVED** [review]
Thanks for the fix!

**Pull request merged**